hdbscan/hdbscan_.py

```diff
@@ -963,5 +979,80 @@ def exemplars_(self):
             return self._prediction_data.exemplars
         else:
             raise AttributeError('Currently exemplars require the use of vector input data'
-                 'with a suitable metric. This will likely change in the '
-                 'future, but for now no exemplars can be provided')
+                                 'with a suitable metric. This will likely change in the '
+                                 'future, but for now no exemplars can be provided')
+
+    @property
+    def relative_validity_(self):
+        if self._relative_validity is not None:
+            return self._relative_validity
+
+        if not self.gen_min_span_tree:
+            raise AttributeError("Minimum spanning tree not present. " +
+                                 "Either HDBSCAN object was created with " +
+                                 "gen_min_span_tree=False or the tree was " +
+                                 "not generated in spite of it owing to " +
+                                 "internal optimization criteria.")
+            return
+
+        labels = self.labels_
+        noise_size, *cluster_size = np.bincount(labels + 1)
```

Comment:
```suggestion
        sizes = np.bincount(labels + 1)
        noise_size = sizes[0]
        cluster_size = sizes[1:]
```
Sorry, I should have looked more closely earlier. We do, in fact, need python 2 compatibility still right now.